bucket4j-spring-boot-starter/pom.xml

```diff
@@ -45,6 +45,10 @@
 			<artifactId>spring-cloud-starter-gateway</artifactId>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>

```

Comment:
The README should be updated to clarify that this starter for the Annotation support. It is not a requirement for the filter usage. Alternatively we have to provide separated starters for Servlet Filter, Webflux, AOP and so on but for now this dependency must be removed.